tests/aeroval/test_experiment_output.py

```diff
@@ -295,37 +261,6 @@ def test_Experiment_Output_clean_json_files_CFG1_INVALIDOBS(eval_config: dict):
     assert len(modified) == 13
 
 
-@pytest.mark.parametrize(
-    "add_names,order,result",
-    [
-        (["c", "b", "a"], None, ["a", "b", "c"]),
-        (["c", "b", "a"], ["c", "b", "a"], ["c", "b", "a"]),
-        (["c", "b", "a"], [42], ["a", "b", "c"]),
-        (["c", "b", "a"], ["b"], ["b", "a", "c"]),
-        (["c", "b", "a"], ["b", "c"], ["b", "c", "a"]),
-    ],
-)
-def test_ExperimentOutput_reorder_experiments(
```

Comment:
This functionality is only called from tests within pyaerocom. If it is required externally, let me know.